platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ConfigurationMigrationService.java

```diff
@@ -36,4 +37,11 @@
      * @return MigrationWarning returned if there were non-fatal issues when exporting
      */
     Collection<MigrationWarning> export(@NotNull Path exportDirectory) throws MigrationException;
+
+    /**
+     * Get a collection of the migratables available for export
+     *
+     * @return Collection of type Describable
+     */
+    Collection<Describable> getInfo();
```

Comment:
Yeah, I considered that. But what we're looking for right now is only the list of _data_  migratables (the optional ones if you will). So if we go with `getMigratablesInfo()` right now and ever need to return all the `Migratables`, we'll have a naming problem.